.github/workflows/fluent-bundle.yml

```diff
@@ -0,0 +1,22 @@
+name: "@fluent/bundle"
+on: [push, pull_request]
```

Comment:
I'd restrict these, similar to what we've done in `python-fluent`. That way it's obvious what should happen on pushes to `gh-pages`. Or maybe just exclude `gh-pages`? I haven't figured out how much the action config of one branch blends others. Maybe not even remotely as much as I thought it would.

Also, do you want to start restricting when automation is run for what now or in a follow-up?

Reply:
I pushed a commit which restricts the workflows to pushes to master and PRs against master.